scripts/run_workflow.pl

```diff
@@ -11,14 +13,16 @@
 # creates an INI file, and, finally, executes the workflow.
 
 my @files;
-my ($run_id, $normal_bam, $tumor_bam, $bedpe, $reference, $output_dir);
+my ($normal_bam, $tumor_bam, $bedpe, $reference, $output_dir, $run_id);
+my $date = localtime->strftime('%Y%m%d');
+
 
 # workflow version
 my $wfversion = "2.0.0";
 
 GetOptions (
   "output-dir=s" => \$output_dir,
-  "run-id=s"   => \$run_id,
+  "run-id:s"   => \$run_id,
```

Comment:
if `run-id` is provided, please validate it so that it only contains `[a-zA-Z0-9_-]`